.travis.yml

```diff
@@ -37,3 +37,16 @@ script:
 
 notifications:
   email: false
+
+before_deploy:
+  ./scripts/travis-ci/create-release-tarball.sh
+
+deploy:
+  provider: releases
+  api_key:
+    secure: JGvlz3E89uDsv/CMspHrFvkGUyc7Bs+qEFYIt01V5DCJ4DIc7KDiAt2zE8KHrw8M5Q5sXkbs6kQfvuB+XY0aMP73vlHdKzg+ZNpUu+Ikmt8uOt8UYZtQQkwOZcD2hsTj/1SzDb+cARQEW7hLDQrp5qL9y6cuPS9aplusY+qaepHPPixGgROpZ/uuj/mtFYJwdN++RS5H33KqhkrDGjbtJo0j1j78l5nSoeXIxqJ9cE2pf2Cb8CZsN8wEiJycLKbI3LKitc7WCODNTvwk2SKUTYm3F8WtQjfBbbghjHn6zFOXucMX1raRJ40LpLNEvd3/3SENWIL0QRF5PQcudZjh9G8Y2pHcSky+rFE9/wCejOKj3Ug54kj9Hsa3Uyqjlyyk0v6xNT8NVpCZsDCstrhuhcHgdWNBBPGQyNc9O+DxVu+VMFUFXFc8lXzLWE+nePBv01WDlv0ywPoiLNN/j68FZNrgbIfsX+56jqQY1Rh3FCtsGpWsYkixQsUsR4hHqdG+9atZGMRmAITksWTMsjVapBZxVf0PB1MuCQSsobTMdi/p+XQXTgDfEucy3qTJNjWFj7oDg8DD+AnsCwwjHMl6FO6loTK5k+ji7ArRHKXxIpm977NkdFGzLCUc2M2/SJ3EylJjadfv43gNgZOz4hje8NJ43kgxKS0r3naVIqppa3I=
```

Comment:
This token will need to be replaced with one generated by the repository owner.

How to do this:
* Install the [Travis CLI](https://github.com/travis-ci/travis.rb).
* Run the following command:
```
$ travis setup releases --force
```
* Follow the instructions. You will be asked for your Github username and password.
* Check the `.travis.yml` file. It will be updated with the new encrypted key in the `deploy` section.
* The file will be mangled, so don't commit it, just copy the new encrypted key, undo all the changes (`git checkout .travis.yml`) and paste the new key in the original file.